client/src/components/_common/Button/Button.jsx

```diff
@@ -65,38 +79,18 @@ const Button = ({
   }
   /* eslint-enable no-console */
 
-  const buttonRootClass = styles['root'];
-
-  let buttonTypeClass;
-  if (type === 'link') {
-    buttonTypeClass = styles['as-link'];
-  } else if (['primary', 'secondary', 'tertiary', 'active'].includes(type)) {
-    buttonTypeClass = styles[`${type}`];
-  } else if (type === '') {
-    buttonTypeClass = type;
-  }
-
-  let buttonSizeClass;
-  if (size === 'small') {
-    buttonSizeClass = styles['size-small'];
-  } else {
-    buttonSizeClass = styles[`width-${size}`];
-  }
-
-  const buttonLoadingClass = isLoading ? styles['loading'] : '';
-
   return (
     <button
       className={`
-        ${buttonRootClass}
-        ${buttonTypeClass}
-        ${buttonSizeClass}
-        ${buttonLoadingClass}
-        ${className}
```

Comment:
For Wes, removed className prop occurred here.

Reply:
Thanks. Fixed in https://github.com/TACC/Core-Portal/commit/8e0a781255095e553d807d8441a754006291107b for https://github.com/TACC/Core-Portal/pull/675.